R/hms_class.R

```diff
@@ -19,19 +21,21 @@ setClass("MetaepochSnapshot", slots = c(
   is_evolutionary = "logical"
 ))
 
-#' hms
-#'
-#' @slot root_id character.
-#' @slot metaepoch_snapshots list.
-#' @slot best_fitness numeric.
-#' @slot best_solution numeric.
-#' @slot total_time_in_seconds numeric.
-#' @slot total_metaepoch_time_in_seconds numeric.
-#' @slot metaepochs_count numeric.
-#' @slot deme_population_sizes numeric.
-#' @slot lower numeric.
-#' @slot upper numeric.
-#' @slot call language.
+#' A S4 class representing a result of hms.
+#'
+#' @slot root_id character - UUID of a root Deme.
+#' @slot metaepoch_snapshots list of objects of class MetaepochSnapshot.
+#' @slot best_fitness numeric - best fitness value of all metaepochs..
+#' @slot best_solution numeric -  best solution of all metaepochs.
+#' @slot total_time_in_seconds numeric - time of a hms execution in seconds.
+#' @slot total_metaepoch_time_in_seconds numeric - time of all metaepochs in seconds.
+#' @slot metaepochs_count numeric - total number of all metaepochs.
+#' @slot deme_population_sizes numeric .
```

Comment:
To w sumie nie jest takie oczywiste - można opisać, że to jest kolekcja, zawierająca potencjalnie różne wielkości populacji na poszczególnych poziomach drzewa.